day1.md

```diff
@@ -0,0 +1,119 @@
+## Day1 Mon. 09/05/16 [Tokyo 東京]
+---
+
+06:30         [Landed @NRT Airport](#landed)<br />
+09:00         [Tokyo Station](#tokyo_station)<br />
+10:00 - 12:00 [Food & Post Office](#first_meal)<br />
+12:30 - 13:40 [Ikebukuro Station & Airbnb place](#ikebukuro_station)<br />
+15:40 - 20:30 [Odaiba Onsen](#odaiba_onsen)
+
+---
+After we <a name="landed">landed</a> around 06:30 am, we picked up our JR passes<sup>[1](#jrpass)</sup> and pocket Wifi<sup>[2](#pocketWifi)</sup> and got some cash<sup>[3](#cash)</sup> from an ATM at the Narita International Airport. Then we took the Narita Express, covered by our JR Passes, to get to Tokyo!
+
+Trains in Japan are amazing! They are extremely clean, like most things in Japan, and comfortable. I'm not exaggerating when I say Shinkansen is more enjoyable than airplanes.
+
+<img src="day1/IMG_20160904_204924.jpg" width="500"><br/>
+
+
+I enjoyed the view along our ride very much. The sky was blue and clear. We passed some fields and saw a lot of typical, Japanese styled, small houses.
+
+We were a bit tired, after the 13-hours flight, but also excited and ready for the awesome trip ahead of us! Although both Matt and I were a bit hungry after we landed, we decided to hold our horses and wait till we got to Tokyo to get our first meal in Japan. "The first meal must to be legit and is worth waiting for!" I said to myself.
+
+After an hour-long ride, we finally arrived at the <a name="tokyo_station">Tokyo station</a>. It was about 9 am which was apparently a rush hour for people to commute to work. The station was fairly crowded. We saw tons of food shops: different kinds of hot steamed rice, bento box meals, and sweets everywhere!
+
+We had been starving and this led us to believe there were a lot of great restaurants outside of the station waiting for us. The station was huge; it took us quite a while to figure out how to get out. Long story short, after resisting all the temptations from foods inside the station and stumbling our ways through, we finally got out and there was nothing but office buildings and a mall with restaurants opening in about two hours.
```

Comment:
Might want another `<br />` at the end here so the last line isn't in line with the next image.